test/spec/transmissions.spec.js

```diff
@@ -1,39 +1,116 @@
 var chai = require('chai')
   , expect = chai.expect
-  , proxyquire = require('proxyquire')
   , sinon = require('sinon')
   , sinonChai = require('sinon-chai')
-  , configuration = require('../../lib/configuration')
-  , MockRequest = require('../mocks/request.js');
+  , nock = require('nock')
+  , SparkPost = require('../../lib/index');
 
 chai.use(sinonChai);
 
 describe('Transmissions Library', function() {
-  var transmission;
+  var client, transmission;
```

Comment:
I can do that. I just put it to the same line because we were only declaring 2 variables with no assignments. If this is our coding standards, I can change it and follow it going forward.